packages/react-components/react-utilities/src/utils/isHTMLElement.ts

```diff
@@ -7,11 +7,76 @@
  * might be problematic while operating with [multiple realms](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms)
  *
  */
-export function isHTMLElement(element?: unknown): element is HTMLElement {
+export function isHTMLElement<ConstructorName extends HTMLElementConstructorName = 'HTMLElement'>(
```

Comment:
It would be nice to add some tests for this function since it currently has none.

Reply:
That's kind of hard though. I'm not sure how to simulate different realms on jest test environment.

Reply:
I think what you added is good. Thanks!